content/en/docs/hertz/tutorials/basic-feature/graceful-shutdown.md

```diff
@@ -23,3 +23,16 @@ If you want to register the `hook` function, you can do so by getting the `Engin
 ```go
 h.Engine.OnShutdown = append(h.Engine.OnShutdown, shutDownFunc)
 ```
+
```

Comment:
Does the way in which the maximum waiting time is adjusted need to be specified?